src/main/java/com/blackducksoftware/integration/jira/common/BlackDuckDataHelper.java

```diff
@@ -116,6 +123,35 @@ public ProjectVersionWrapper getProjectVersionWrapper(final VersionBomComponentV
         return new ProjectVersionWrapper();
     }
 
+    public Optional<NotificationType> getNotificationTypeFromPolicyRule(final PolicyRuleView policyRule) {
+        final Optional<PolicySummaryStatusType> optionalPolicySummaryStatusType = getPolicySummaryStatusTypeFromRule(policyRule);
+        if (optionalPolicySummaryStatusType.isPresent()) {
+            switch (optionalPolicySummaryStatusType.get()) {
+                case IN_VIOLATION:
+                    return Optional.of(NotificationType.RULE_VIOLATION);
+                case NOT_IN_VIOLATION:
+                    return Optional.of(NotificationType.RULE_VIOLATION_CLEARED);
+                case IN_VIOLATION_OVERRIDDEN:
+                    return Optional.of(NotificationType.POLICY_OVERRIDE);
```

Comment:
Are we ignoring the other notification types here on purpose? Like BOM_EDIT?

Reply:
Yes, this is just for PolicyRules.